Makefile

```diff
@@ -19,6 +19,8 @@ environment:
 	poetry config virtualenvs.in-project true 
 	poetry install
 	@echo 🚨 Be sure to add poetry to PATH
+	wget -O sample-data.zip https://github.com/microsoft/electionguard/releases/download/v0.95.0/sample-data.zip
+	unzip sample-data.zip
```

Comment:
you might want to add a -o to this to allow the files to overwrite a previous download.  This is what is happening on the cpp project as well.